tests/javascript/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "joomla-cms",
+  "version": "1.0.0",
+  "description": "Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cms/readme)](https://github.com/igrigorik/ga-beacon)\r ====================",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Thanuditha Ruchiranga Wickramasinghe",
+  "license": "GNU",
```

Comment:
Use GPL-2.0 here (to match joomla-cms) - see https://spdx.org/licenses/ and licenses section in npm